server/backend/rpcserver.go

```diff
@@ -47,6 +47,16 @@ func New(db *gorm.DB) *CampusServer {
 	}
 }
 
+func (s *CampusServer) GetAccessPoint(ctx context.Context, in *pb.APRequest) (*pb.AccessPoint, error) {
+	log.Printf("Received request for AP with name: %s and timestamp: %s", in.Name, in.Timestamp)
+	return &pb.AccessPoint{Name: "response"}, nil
+}
+
+func (s *CampusServer) ListAccessPoints(in *pb.APRequest, stream pb.Campus_ListAccessPointsServer) error {
+	log.Println("Requesting list")
+	return nil
+}
+
```

Comment:
I assume these will be implemented later?

Reply:
done.